RawCMS.Library/BackgroundJobs/HangfireAuthorizationFilter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Hangfire.Dashboard;
+
+namespace RawCMS.Library.BackgroundJobs
+{
+    public class HangfireAuthorizationFilter : IDashboardAuthorizationFilter
+    {
+        public bool Authorize(DashboardContext context)
+        {
+            var httpContext = context.GetHttpContext();
+            return true;// httpContext.User.Identity.IsAuthenticated; //TODO: add authentication role
```

Comment:
Should this be httpContext.User.Identity.IsAuthenticated be used? Or is the todo to add authentication role so that IsAuthenticated will work.